### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by M-Grisha**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,21 @@
...
+  if (days >= 7) {
+    return Number(days) * 40 - 50;
+  }
+
+  if (days >= 3) {
+    return Number(days) * 40 - 20;
+  }
+
+  if (days < 3) {
+    return Number(days) * 40;
```
Comment (tetiana-ivanchenko) [inline src/calculateRentalCost.js:44]: move "magic" numbers to variables.
I mean when someone looks at your code it's not clear what is 40, 20, 50?
also you can create a variable for days * 40.
and why do you need a Number(days)? days is already a number

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,26 @@
...
 function calculateRentalCost(days) {
-  // write code here
+  const carCost = 40;
+  const sevenDaysDiscount = 50;
+  const threeDaysDiscount = 20;
+  const rentCost = days * carCost;
+
+  if (days === 0) {
+    return 0;
+  }
```
Comment (tetiana-ivanchenko) [inline src/calculateRentalCost.js:38]: redundant

File: src/calculateRentalCost.js [hunk 3]
```diff
@@ -28,7 +28,26 @@
...
+    return rentCost - sevenDaysDiscount;
+  }
+
+  if (days >= 3) {
+    return rentCost - threeDaysDiscount;
+  }
+
+  if (days < 3) {
+    return rentCost;
+  }
```
Comment (tetiana-ivanchenko) [inline src/calculateRentalCost.js:50]:
```suggestion
  return rentCost;
```

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

File: src/calculateRentalCost.js (hunk 3 above)
Comment (AlisaVlasova) [reply src/calculateRentalCost.js:46]: цю умову можна видалити, всеодно повертається rentCost
`  if (days < 3) {
    return rentCost;
  }`

**Review by AlisaVlasova — CHANGES_REQUESTED** [review]

**Comment by M-Grisha** [comment]
Tnx, I fix and understand my mistakes)

**Review by tetiana-ivanchenko — APPROVED** [review]

**Pull request closed without merging**